src/Microsoft.ML.AutoML/API/BinaryClassificationExperiment.cs

```diff
@@ -90,11 +90,6 @@ public enum BinaryClassificationMetric
     /// </summary>
     public enum BinaryClassificationTrainer
     {
-        /// <summary>
-        /// See <see cref="AveragedPerceptronTrainer"/>.
-        /// </summary>
-        AveragedPerceptron,
```

Comment:
Why are you removing these?

Reply:
Because AveragedPerceptron is no longer a candidate binary trainer. We can add it back if we want to re-add it again